ui/app/AppLayouts/Profile/views/BrowserView.qml

```diff
@@ -76,12 +80,8 @@ SettingsContentBase {
                 title: qsTr("Show Favorites Bar")
                 components: [
                     StatusSwitch {
-                        checked: localAccountSensitiveSettings.shouldShowFavoritesBar
-                        onCheckedChanged: {
-                            if (localAccountSensitiveSettings.shouldShowFavoritesBar !== checked) {
-                                localAccountSensitiveSettings.shouldShowFavoritesBar = checked
-                            }
-                        }
+                        checked: accountSettings.shouldShowFavoritesBar
+                        onToggled: { accountSettings.shouldShowFavoritesBar = checked }
```

Comment:
I'm not sure if `checked` property was changed before `toggled` signal, so basically, its equal to `accountSettings.shouldShowFavoritesBar = accountSettings.shouldShowFavoritesBar`